tests/network/sriov.go

```diff
@@ -86,11 +86,6 @@ var _ = Describe("SRIOV", Serial, decorators.SRIOV, func() {
 
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
-
-		// Check if the hardware supports SRIOV
-		if err := validateSRIOVSetup(virtClient, sriovResourceName, 1); err != nil {
```

Comment:
There's another occurence of validateSRIOVSetup if you want to delete that and the function itself

Reply:
https://github.com/kubevirt/kubevirt/blob/b99642973573a57ea2a4e6e0d8d23910e9b7fe8c/tests/network/sriov.go#L254-L259

Reply:
Thanks for the heads up! 

I pushed new changes replacing the Skip() with Fail()